src/components/editor/faultTree/Editor.tsx

```diff
@@ -190,35 +195,55 @@ const Editor = ({ setAppBarName }: DashboardTitleProps) => {
     const result = list.reduce((maxItem, currentItem) => {
       return currentItem.probability > maxItem.probability ? currentItem : maxItem;
     });
-    return result
+    return result;
```

Comment:
@blcham it is weird that it was not captured by pre-commit before, it might be broken

Reply:
is there any problem ?

Reply:
I am just wondering why this formatting issue was not captured by our pre-commit workflow, this is not a problem linked to this issue or PR :)